client/src/pages/UserSettings.js

```diff
@@ -0,0 +1,270 @@
+import React, { useState, useEffect } from "react";
+import { ChallengeOptions } from "../components/Form/ChallengeDropdown";
+import { Grid, Image, Form } from 'semantic-ui-react';
+import ReadGroup from "../components/Form/readAccount"
+import EditGroup from "../components/Form/editAccount"
+import API from "../utils/API";
+
+function Settings() {
+
+  const [nameState, setNameState] = useState("Read");
+  const [emailState, setEmailState] = useState("Read");
+  const [passState, setPassState] = useState("Read");
+
+  const [userName, setUserName] = useState("");
+  const [userEmail, setUserEmail] = useState("");
+  const [userPass, setUserPass] = useState("");
+
+  const [nameInput, setNameInput] = useState("")
+  const [emailInput, setEmailInput] = useState("")
+  const [passInput, setPassInput] = useState("")
+
+  const [placeholder1, setPlaceholder1] = useState("");
+  const [placeholder2, setPlaceholder2] = useState("");
+  const [placeholder3, setPlaceholder3] = useState("");
+
+  const [challenges, setChallenges] = useState("")
+
+  const [challCat1, setChallCat1] = useState(challenges[0]);
+  const [challCat2, setChallCat2] = useState(challenges[1]);
+  const [challCat3, setChallCat3] = useState(challenges[2]);
+
+  const id = "5fa74a6f9d0fcaeb93ac934e"
+
+  const getUserSettings = () => {
+    API.getUserSettings(id)
+      .then((res) => {
+        const c1 = res.data[0].challengeCategories[0]
+        const c2 = res.data[0].challengeCategories[1]
+        const c3 = res.data[0].challengeCategories[2]
+        console.log(res);
+        setUserName(res.data[0].name);
+        setUserEmail(res.data[0].email);
+        setUserPass(res.data[0].password);
+        setChallenges(res.data[0].challengeCategories)
+        if (!c1) {
+          setPlaceholder1("Choose Category");
+        } else {
+          setPlaceholder1(c1)
+        }
+
+        if (!c2) {
+          setPlaceholder2("Choose Category");
+        } else {
+          setPlaceholder2(c2)
+        }
+
+        if (!c3) {
+          setPlaceholder3("Choose Category");
+        } else {
+          setPlaceholder3(c3)
+        }
+
+      })
+  }
+
+  useEffect(() => {
```

Comment:
what are these doing and do we want to put them in main?

Reply:
I'll remove those